packages/web-components/fast-foundation/src/anchor/anchor.ts

```diff
@@ -80,6 +80,20 @@ export class Anchor extends FASTElement {
      */
     @attr
     public type: string;
+
+    /**
+     *
+     * Default slotted content
+     *
+     * @public
```

Comment:
Same question as above here, should this be a public facing API or is this an implementation detail of the component? If it is the latter, this should be marked as `@internal`